PushNotifications.md

```diff
@@ -326,6 +332,12 @@ Users can listen to messages in each platform using the native message listeners
 
 Take a look at the example app platform specific code to handle messages. For iOS, this is `AppDelegate.m`, and in Android, it is `PushMessagingService.java`. For further help on implementing the Platform specific message handlers, see "On Android" and "On iOS" sections on [Push Notifications - Device activation and subscription](https://ably.com/documentation/general/push/activate-subscribe).
 
+### Additional considerations and resources
+- For tips on how best to use push messaging on Android, read [Notifying your users with FCM](https://android-developers.googleblog.com/2018/09/notifying-your-users-with-fcm.html). For example:
+  - Show a notification to the user as soon as possible without any additional data usage or processing. Perform additional synchronization work asynchronously after that, using [workmanager](https://pub.dev/packages/workmanager). You can also update the notification with a nicer notification, such as with buttons.
```

Comment:
Could we say "You can also update the notification to be more visually appealing. For example, displaying buttons to enable the user to interact with it." (Trying to get rid of two occurrences of the word "notification" in the same sentence and also the word "nicer" 😉 )

Reply:
I've updated that section but avoiding the visually appealing thing, because its not for visual appeal but for more user interaction: 47be1e6c4eb1be3ae8186546130c4cfde82dfaa6

Reply:
👍